app/views/shared/_my_org.html.erb

```diff
@@ -0,0 +1,20 @@
+<% others = allow_other_orgs ||= false %>
+
+<%= f.label :org_name, _('Organisation'), class: 'control-label' %>
+<%= render partial: "shared/accessible_combobox",
+             locals: {name: "user[org_name]",
+                      id: "user_org_name",
+                      default_selection: default_org,
+                      models: orgs,
+                      attribute: 'name'} %>
+
+<% if others %>
```

Comment:
We can join this if statement with the one below as they both check the same condition